app/routes/ffxiv.weekly-price-group-delta.tsx

```
@@ -0,0 +1,628 @@
+                <span className="text-gray-700 dark:text-gray-200">
+                  HQ Only
+                </span>
+              </label>
+            </div>
+
+            <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+              <div>
+                <label className="block text-sm font-medium text-gray-700 dark:text-gray-200 mb-1">
+                  Region
+                </label>
+                <select
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)